### DISCUSSION TITLE
ReadTimeout occurred. Retrying...

### DISCUSSION THREAD

**Discussion by MCancemi** [omkarcloud/google-maps-scraper#52] [Q&A]
Hello, I'm having problems after a few minutes of scrapping, it gives me this error:

Filtered 5 links from 5.
View written JSON file at output/filename.json
View written CSV file at output/filename.csv
ReadTimeout occurred. Retrying...
ReadTimeout occurred. Retrying...
ReadTimeout occurred. Retrying...
ReadTimeout occurred. Retrying...
ReadTimeout occurred. Retrying...
Max retries exceeded. Unable to fetch IP details.

**Comment by Chetan11-dev** [comment]
Bug has been fixed, Please run ` python -m pip install bose --upgrade` to solve this problem.

**Reply by MCancemi** [reply]
Thanks so much!

**Reply by Chetan11-dev** [reply]
Welcome
